### PR TITLE
Add first and second tasks

### PR DESCRIPTION
**Pull request by EgorAntonovich**
First task about create different types variables.
Second task about fix type of variable.

### CODE REVIEW

File: TypeScript/task1_BasikTypes.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export var var1Boolean: boolean = true;
+// TODO:
+export var var2Decimal: number = 13;
+export var var3Hex: number = 0xf00d;
```
Comment (DmitryEvmenov) [inline TypeScript/task1_BasikTypes.ts]: What does `0x` prefix mean? The original task says to set value to `f00d`, why do we need to add it?

File: TypeScript/task1_BasikTypes.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export var var1Boolean: boolean = true;
+// TODO:
```
[the remark this answers is not in the document]
Comment (EgorAntonovich) [reply TypeScript/task1_BasikTypes.ts]: In task after it i had to add my variables.

File: TypeScript/task1_BasikTypes.ts (hunk 1 above)
Comment (EgorAntonovich) [reply TypeScript/task1_BasikTypes.ts]: Yep, it's means that  number would be set as hex. Like a binary with 0b prefix or octal with 0o prefix.

File: TypeScript/task1_BasikTypes.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+export var var7Array = [1, 'test', {a: 3}, 4, 5];
+export var var8NumericArray: number[] = [1,2,3,4,5];
+export var var9Tuple: [string, number] = ['key', 12345];
+export enum Color {Red = 1, Green = 2, Blue = 4};
+export var var10Enum: number = Color.Blue;
+export var var11ArrayOfAny: Array<any> = [1, 'test', {a: 3}, 4, 5];
+export var var12VoidFunction = function(): void {};
+export var var13Null = null;
+export var var14Undefined = undefined;
+export var var15NeverFunction = function(): never{throw new Error()};
```
[the remark this answers is not in the document]
Comment (EgorAntonovich) [reply TypeScript/task1_BasikTypes.ts]: we can use void in this situation, but in typescript  type "never" use in functions which have an endless cycle or return a exception.

**Pull request merged**